primitives/src/types/jobs.rs

```diff
@@ -306,23 +388,18 @@ pub struct DKGSignatureResult {
 
 #[derive(PartialEq, Eq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub struct ZkSaasPhaseOneResult {
+pub struct ZkSaasCircuitResult {
 	/// The job id of the job
 	pub job_id: JobId,
 
 	/// List of participants' public keys
 	pub participants: Vec<Vec<u8>>,
-
-	/// The data to verify
-	pub data: Vec<u8>,
 }
 
 #[derive(PartialEq, Eq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub struct ZkSaasPhaseTwoResult {
-	/// The data to verify
-	pub data: Vec<u8>,
-
-	/// The expected key for the signature
-	pub signing_key: Vec<u8>,
+pub struct ZkSaasProofResult {
```

Comment:
I would suggest breaking out the types into more files to further separate the TSS/DKG from the zkSaaS.